docs/reference/traits.rst

```diff
@@ -0,0 +1,12 @@
+.. _traits:
+
+========
+Traits
+========
+
+When developing new hybrid components, code should enforce trait verification
```

Comment:
Maybe just mention that all existing blocks in Hybrid implement state traits constraints...

Reply:
👍